src/gfnff/gfnff_ini.f90

```diff
@@ -0,0 +1,1820 @@
+! This file is part of xtb.
+!
+! Copyright (C) 2019-2020 Stefan Grimme
+!
+! xtb is free software: you can redistribute it and/or modify it under
+! the terms of the GNU Lesser General Public License as published by
+! the Free Software Foundation, either version 3 of the License, or
+! (at your option) any later version.
+!
+! xtb is distributed in the hope that it will be useful,
+! but WITHOUT ANY WARRANTY; without even the implied warranty of
+! MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+! GNU Lesser General Public License for more details.
+!
+! You should have received a copy of the GNU Lesser General Public License
+! along with xtb.  If not, see <https://www.gnu.org/licenses/>.
+
+subroutine gfnff_ini(pr,makeneighbor,mol,ichrg)
+      use xtb_mctc_accuracy, only : wp
+      use xtb_type_molecule
+      use gff_param
+      use gff_d3com, only: rcov,r2r4
+      use gff_frag_hess
+      use xtb_restart
+      implicit none
+!--------------------------------------------------------------------------------------------------
+      type(TMolecule), intent(in) :: mol   ! # molecule type
+
+      integer, intent(in) :: ichrg         ! mol. charge
+      logical, intent(in) :: pr            ! print flag
+      logical, intent(in) :: makeneighbor  ! make a neigbor list or use existing one?
+!--------------------------------------------------------------------------------------------------
+
+      integer ati,atj,atk,i,j,k,l,lin,nn,ii,jj,kk,ll,m,rings,ia,ja,ij,ix,nnn,idum,ip,ji,no
+      integer ineig,jneig,nrot,bbtyp,ringtyp,nn1,nn2,hybi,hybj,pis,ka,nh,jdum,hcalc,nc
+      integer ringsi,ringsj,ringsk,ringl,npi,nelpi,picount,npiall,maxtors,rings4,nheav
+      integer nm,maxhb,ki,n13,current,ncarbo,ctype,mtyp1,mtyp2
+      integer ind3(3),sr(20),cr(10,20),niel(86)
+      integer qloop_count,nf,nsi,nmet,nhi,nhj,ifrag
+      integer hbA,hbH,Bat,atB,Aat,Hat
+      integer AHB_nr
+      integer bond_hbn
+      interface
+         integer function itabrow6(i)
+            integer i
+         end function
+      end interface
+
+      real(wp) r0,pi,ff,omega,f1,f2,phi,valijklff,ringf,fcn
+      real(wp) shift,dum,dum1,dum2,dum4,qafac,fqq,feta
+      real(wp) sumppi,fpi,fxh,fijk,fsrb2,ees
+      real(wp) fheavy,fn,eold,fctot,fij
+      real(wp) hbpi(2),hbpj(2),sdum3(3)
+      real(wp) bstrength
+      real(wp) xx(20)
+      real(wp) fkl,qreps,fbsmall,bohr
+
+      parameter (pi=3.1415926535897932384626433832795029_wp)
+      parameter (bohr=1.0_wp/0.52917726_wp)
```

Comment:
Definitely needs to get replaced with `xtb_mctc_convert`

Reply:
same here